docs/java-application-development/development-environment-setup.md

```diff
@@ -1,183 +1,161 @@
 
-Finally, click **Finish** to import the projects.  At this point, you should have four projects in your workspace.  The four projects are as follows:
+If you plan to contribute to Eclipse Kura you might want to [create a fork](https://docs.github.com/en/get-started/quickstart/fork-a-repo), see our [contributing guide](https://github.com/eclipse/kura/blob/develop/CONTRIBUTING.md) for further informations. For the purpose of this tutorial we'll work with a fictional fork for the username `user`. To clone the repo use the link appropriate for your fork, in our case it will be: `https://github.com/user/kura.git`
 
-* org.eclipse.kura.api – the core Kura API.
+Keep in mind that the "Root install folder" is where the Eclipse executable will be installed and the Kura sources will be downloaded (in the `git` subfolder).
 
-* org.eclipse.kura.demo.heater – an example project that you can use as a starting point for creating your own bundle.
+![](./images/development-environment-setup/devenvsetup-4.png)
 
-* org.eclipse.kura.emulator – the emulator project for running Kura within Eclipse (Linux/Mac only).
+Press Next, leave all Bootstrap Tasks selected and press the Finish button
 
-* target-definition – a set of required bundles that are dependencies of the APIs and Kura.
+![](./images/development-environment-setup/devenvsetup-5.png)
 
-    ![Import Project](./images/development-environment-setup/image6.png)
+Accept all the licenses and wait for the installation to finish. 
 
-Eclipse will also report some errors at this point.  See the next section to resolve those errors.
+![](./images/development-environment-setup/devenvsetup-6.png)
 
-### Workspace Setup
-This section will guide the users to configure the development workspace environment. 
+At first startup Eclipse IDE will checkout the code, perform a full build and configure a few Working Sets
 
-#### JRE Configuration
-The latest Eclipse IDEs require and configure, by default, a Java 11 environment.
-In order to be able to leverage and develop using the new workspace for Kura, the user will be required to perform a one-time operation to specify to the IDE a Java 8 JDK.
-Opening the Eclipse preferences and selecting the Installed JREs in the Java section, the user has to select an installed Java 8 instance. 
+![](./images/development-environment-setup/devenvsetup-7.png)
 
-![Set JDK 8](./images/development-environment-setup/image7.png)
+![](./images/development-environment-setup/devenvsetup-8.png)
 
-After applying the configuration change, the user will be prompted to align also the compiler options. To do so, selecting the Compiler entry in the Java section, the user has to select 1.8 from the list of available Java versions.
+When the tasks are completed go to into the Package Explorer and Target Platform > Target-Definition > Kura Target Platform Equinox 3.16.0, and press "Set as Target Platform" located at the top right of the window:
```

Comment:
> When the tasks are completed go to into the Package

When the tasks are completed, go to into the Package

Reply:
Addressed in f6b557810bfc0d1c70f289e47531f2a948ff5041